apps/server/src/modules/auth/next-auth-options.ts

```diff
@@ -98,7 +98,7 @@ export const NextAuthOptionsProvider: FactoryProvider<NextAuthOptions> = {
       adapter: prismaAdapter,
       debug: !config.node.prod,
       session: {
-        strategy: config.node.prod ? 'database' : 'jwt',
+        strategy: 'jwt',
```

Comment:
Also uses `jwt` session token on production.

Reply:
It seems safe; only increase the cookie size, which is acceptable